src/github.com/matrix-org/dendrite/clientapi/readers/profile.go

```diff
@@ -159,3 +196,49 @@ func getLocalPart(userID string) string {
 	// Return the part after the "@"
 	return strings.Split(username, "@")[1]
 }
+
+// Send an update using kafka to notify the roomserver of the profile update
+// Returns an error if the update failed to send
+func sendUpdate(userID string, oldProfile *authtypes.Profile, newAvatarURL string,
+	newDisplayName string, accountDB *accounts.Database, cfg config.Dendrite,
```

Comment:
That comes from an older version of this function, I forgot to remove it in the parameters.